index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+
+<html>
+  <head>
+    <title>Erica Hendricks</title>
+    <meta name="viewport" content="width:display-width, initial-scale=1">
```

Comment:
Typo here: "display" should be "device". I recommend that you get used to developing with your Developer Tools console window open. For this typo, the console shows a warning along with the line number of the problem.